src/components/Stats/Volume/Stats.volume24.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { withKnobs } from '@storybook/addon-knobs'
+import Volume from './Volume'
+
+storiesOf('position/Stats', module)
+  .addDecorator(withKnobs)
+  .add('volume24', () => {
+    const [data, seData] = React.useState<Array<{ timeStamp: number; value: number }>>([])
+
+    const timeSlicer = 10
+    let timeCreator = 0
+
+    React.useEffect(() => {
+      const data: Array<{ timeStamp: number; value: number }> = []
+
+      for (let i = 0; timeCreator < 24 * 60; i++) {
+        const hours = ~~(timeCreator / 60)
+        const minutes = ~~(timeCreator % 60)
```

Comment:
~~ is not necessary for modulo operation

Reply:
done